### PR TITLE
Ignore empty <single/>

### PR DESCRIPTION
**Pull request by jayvdb**
Fixes two exceptions in the test suite.

### CODE REVIEW

File: citeproc/model.py [hunk 1]
```diff
@@ -756,7 +756,10 @@ def _term(self, item):
         if plural:
             text = term.multiple
         else:
-            text = term.single
+            try:
+                text = term.single
+            except Exception as e:
+                text = None
```
Comment (brechtm) [inline citeproc/model.py]: We should [limit which exceptions are caught](https://www.quantifiedcode.com/knowledge-base/correctness/Avoid%20untyped%20exception%20handlers/3JwOg9ad). AttributeError in this case, I believe.

But in this case it's probably better to check whether `term` is equal to `None`.

**Comment by brechtm** [comment]
The root cause here is that term equals None. If the plural form is requested, that will fail too. We should find out why term is None.

**Comment by brechtm** [comment]
The CSL style in `bugreports_GreekStyleProblems` needs the 'verb-short' form of the 'volume' term. This form isn't available in the el-GR locale, but neither in the en-GB or en-US locales. Perhaps CSL requires falling back to the standard form (the term in the locale where `form` not specified)?

**Comment by brechtm** [comment]
Relevant text in the CSL documentation ([Terms](http://docs.citationstyles.org/en/stable/specification.html#terms)):

> If a style uses a term in a form that is undefined (even after [Locale Fallback](http://docs.citationstyles.org/en/stable/specification.html#locale-fallback)), there is fallback to other forms: “verb-short” first falls back to “verb”, “symbol” first falls back to “short”, and “verb” and “short” both fall back to “long”. If no locale or form fallback is available, the term is rendered as a empty string.

This fallback mechanism is not yet implemented in citeproc-py.

**Comment by jayvdb** [comment]
Not the right solution

**Pull request closed without merging**